l1-verifier/scripts/test.js

```diff
@@ -10,36 +10,43 @@ async function main() {
   const server = ganache.server(options);
   console.log('Starting server');
   const port = await new Promise((resolve, reject) => {
-    server.listen(0, async (err) => {
```

Comment:
Can you revert these changes?

Reply:
When I switched from `fork` to `spawn` the option to pass RPC_PORT dynamically wasn't working so I decided to hardcode the port. Would hardcoding port number cause any issues?

Reply:
Yes; there's no guarantee a fixed port number is available. Why did you need to switch from `fork` to `spawn`?

Reply:
`fork` takes the path to run the command and you were specifying the relative path where `spawn` takes the actual command that runs in the context of the environment

https://nodejs.org/docs/latest-v18.x/api/child_process.html#child_processforkmodulepath-args-options

Reply:
Okay; was it not working? It worked for me and in the build.

Reply:
Which build? This change is already in the `main` branch as part of this PR https://github.com/ensdomains/evmgateway/pull/6

Reply:
I missed that it was in the other pr. Can you please revert it? I still don't understand why you needed to make this change in the first place.